### DISCUSSION TITLE
In Minnesota there are 3 operators without location atributes

### DISCUSSION THREAD

**Discussion by dmitrykinakh** [transitland/transitland#378] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8025637/195603476-fc9ca6a0-a769-4c9c-b61f-08099876e1fa.png)
Could you please let me know how these 3 operators can be searchable by USA state Minnesota if they have no location attributes?
I've checked other states across USA and such issue is not present anywhere else. 
Are these 3 operators just bad data we avoid to use?

**Comment by drewda** [comment, marked as answer]
This is likely a side effect of a feed version from (Minneapolis/St. Paul) Metro Transit accidentally being loaded into (Omaha) Metro Transit the other day. We should be able to clean up the dangling data soon.

**Reply by dmitrykinakh** [reply]
Could you please ping me here once it will be fixed?

**Reply by drewda** [reply]
@dmitrykinakh this should be fixed now.

**Reply by dmitrykinakh** [reply]
![image](https://user-images.githubusercontent.com/8025637/196289156-fb5b517d-c792-4758-8f60-ef5c178f79c5.png)

Is it the expected result?

**Reply by drewda** [reply]
Yes, if you click through to those operator pages, you'll see that both of those have service spanning multiple states, including Minnesota.

Transitland uses heuristics to rank the city/state/country geography is the most representative one for each feed.

**Reply by dmitrykinakh** [reply]
Thanks a lot for the feedback!
